PHC-Android/PHC/src/main/java/phc/android/Checkin/SearchResultsFragment.java

```diff
@@ -171,41 +186,53 @@ public void onAttach(Activity activity) {
     @Override
     public void onResume() {
 
-        //Get the search parameters
+        // Get the search parameters
         SharedPreferences searchPreferences = getActivity().getSharedPreferences(SearchFragment.SEARCH_PARAMETERS, 0);
         final String firstName = searchPreferences.getString("firstName", null);
         final String lastName = searchPreferences.getString("lastName", null);
         final SimpleDateFormat df = new SimpleDateFormat("yyyy-MM-dd");
 
+        // Get userId and authToken
+        mUserPreferences = getActivity().getSharedPreferences(USER_PREFS_NAME,
+                Context.MODE_PRIVATE);
+        final String userId = mUserPreferences.getString("user_id", null);
+        final String authToken = mUserPreferences.getString("auth_token", null);
+
         //If there are search parameters,
-        if(firstName != null && lastName != null) {
-            String url = getActivity().getResources().getString(R.string.request_url);
+        if(firstName != null && lastName != null && userId != null && authToken != null) {
```

Comment:
if the user does not enter a first or last name, will it by default be empty string "" or null or something else?
just wanted to make sure that search will still work if one is missing